proto/src/main/proto/pb.proto

```diff
@@ -677,6 +677,7 @@ message Filter {
     repeated string bannedPrivilegedDevPubKeys = 23;
     bool disable_auto_conf = 24;
     repeated string banned_auto_conf_explorers = 25;
+    repeated string node_addresses_banned_from_network = 26;
```

Comment:
Filter hash won't be backwards compatible so care is needed when this is rolled out

Reply:
Ah true. Damn... We can keep old filter and create a new one. Or use extraMap (not great)....
There will be more addition to filter, so I guess to handle multiple filters is best. Most users update quickly so we do not need to maintain the old filters forever...